### PR TITLE
Feat(eos_designs): Schema for CVP

### PR DESCRIPTION
**Pull request by carlbuchmann**
# EOS Designs Schema for CVP

Schema for the following keys:

- cvp_ingestauth_key
- cvp_instance_ip
- cvp_instance_ips
- cvp_token_file
- terminattr_disable_aaa
- terminattr_ingestexclude
- terminattr_ingestgrpcurl_port
- terminattr_smashexcludes

## Contributor Checklist

- [x] Review documentation for accuracy and completeness and compare with old documentation.
- [x] Ensure `convert_types` is leveraged were applicable to auto-convert, accepting `str` in `int` and `int` in `str`.
- [x] Verify that data model is fully covered in the described schema. Easiest by looking at the generated documentation.
- [x] run `pre-commit run schemas --all`
- [x] Verify that CI pass.

## Reviewer Checklist

**Reviewer 1:** Julio

- [x] Review documentation for accuracy and completeness and compare with old documentation.
- [x] Ensure `convert_types` is leveraged were applicable to auto-convert, accepting `str` in `int` and `int` in `str`.
- [x] Verify that data model is fully covered in the described schema. Easiest by looking at the generated documentation.
- [x] Verify that CI pass.

**Reviewer 2:**

- [ ] Review documentation for accuracy and completeness and compare with old documentation.
- [ ] Ensure `convert_types` is leveraged were applicable to auto-convert, accepting `str` in `int` and `int` in `str`.
- [ ] Verify that data model is fully covered in the described schema. Easiest by looking at the generated documentation.
- [ ] Verify that CI pass.

### CODE REVIEW

File: ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/cvp_instance_ip.schema.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  cvp_instance_ip:
+    documentation_options:
+      filename: Management Settings
+      table: CloudVision Settings
+    type: str
+    description: |
```
Comment (gmuloc) [inline ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/cvp_instance_ip.schema.yml:11]: Maybe in this doc should we encourage to use the `cvp_instance_ips` ?

File: ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/cvp_token_file.schema.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  cvp_token_file:
+    documentation_options:
+      filename: Management Settings
+      table: CloudVision Settings
+    type: str
+    default: "/tmp/cv-onboarding-token"
+    description: CVP token file is path to token file on switch and is only applicable to CV as a Service
```
Comment (gmuloc) [inline ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/cvp_token_file.schema.yml]:
```suggestion
    description: cvp_token_file is the path to the token file on the switch and is only applicable to CV as a Service
```

File: ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/terminattr_ingestexclude.schema.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  terminattr_ingestexclude:
+    documentation_options:
+      filename: Management Settings
+      table: CloudVision Settings
+    type: str
+    default: "/Sysdb/cell/1/agent,/Sysdb/cell/2/agent"
```
Comment (gmuloc) [inline ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/terminattr_ingestexclude.schema.yml:11]: the initial doc had
```
terminattr_ingestexclude: "< ingest excludes | default -> /Sysdb/cell/1/agent,/Sysdb/cell/2/agent >"
```

exclude**S** - should we document how to give multiple ? ( I guess it is comma separated) + the key is kind of inconsistent with smashexclude**S** (probably historic) should we log some issue to align?

File: ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/terminattr_ingestgrpcurl_port.schema.yml [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+keys:
+  terminattr_ingestgrpcurl_port:
+    documentation_options:
+      filename: Management Settings
+      table: CloudVision Settings
+    type: int
+    convert_types:
+    - str
+    default: 9910
+    description: Port number
```
Comment (gmuloc) [inline ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/terminattr_ingestgrpcurl_port.schema.yml]:
```suggestion
    description: Port number for Terminattr ingest GRPC
```

File: ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/cvp_instance_ip.schema.yml (hunk 1 above)
Comment (carlbuchmann) [reply ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/cvp_instance_ip.schema.yml:11]: Should we deprecate this key, then?

Comment (carlbuchmann) [reply ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/cvp_instance_ip.schema.yml:11]: I will move this to an issue and we can address it when we revisit the docs.

File: ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/terminattr_ingestexclude.schema.yml (hunk 3 above)
Comment (carlbuchmann) [reply ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/terminattr_ingestexclude.schema.yml:11]: Yes, let's convert to an issue, and address this in a future PR.

**Review by JulioPDX — APPROVED** [review]
LGTM, Thank you for the schema review!

**Pull request merged**